### DISCUSSION TITLE
dashboard export for sharing externally api

### DISCUSSION THREAD

**Discussion by fgouteroux** [grafana/grafana#58241] [Dashboards, Panels & Visualizations]
Hello,

I'm using the dashboard export for sharing externally to push dashboard from dev environment to preprod/prod environment. 

So I'm using the button Export for sharing externally in Web UI. I notice that this feature is done by [react](https://github.com/grafana/grafana/blob/5361efc225e7d4b06322b0b9b2005102aa786a26/public/app/features/dashboard/components/ShareModal/ShareExport.tsx#L22), and I would like to have the same feature in the API. 

Maybe it could be implemented under`GET /api/dashboards/uid/:uid/externally_sharing` or `GET /api/dashboards/uid/:uid?export=externally_sharing`

**Comment by fgouteroux** [comment]
Same as https://github.com/grafana/grafana/discussions/45167

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
